packages/muze-utils/src/DataSystem/selection.js

```diff
@@ -220,6 +227,15 @@ class Selection {
         return this;
     }
 
+    sort (sortFn) {
+        const updateData = this._updatedata;
+        const dataObjects = this._dataObjects;
+
+        this._updatedata = new Map([...updateData.entries()].sort(sortSelection(dataObjects, updateData, sortFn)));
+        this._dataObjects = new Map([...dataObjects.entries()].sort(sortSelection(dataObjects, updateData, sortFn)));
+        return this;
```

Comment:
Isn't the sortSelection function being called multiple times here? I think it should be called only once